aws_lambda_builders/workflows/nodejs_npm/utils.py

```diff
@@ -0,0 +1,40 @@
+"""
+Commonly used utilities
+"""
+
+import os
+import tarfile
+import subprocess
+import sys
+
+
+class OSUtils(object):
```

Comment:
Do we really need all the wrapping around os.path? We should really move on up into pathlib and operate on objects

Reply:
I just copied this design from the python_pip builder implementation, so I guess this is for a more general discussion. Happy to change it if you decide internally that you want to take a different direction, just let me know

Reply:
I didn't realize we already do this. We can refactor these things later.